File: Assignment_35/Program5.java
import java.util.*;

class Reverse
{
    public static String ReverseString(String str)
    {
        char[] arr = str.toCharArray();
         int i = 0, j = arr.length - 1;
         char temp;

        for(; i < j;  i++ , j--);
        {
     
            temp = arr[i];
            arr[i] = arr[j];
            arr[j] = temp;
        }
        return new String(arr);
    }
}
class Program5
{
    public static void main(String[] args) 
    {
         

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String :");
        String str = sobj.nextLine();

        Reverse robj = new Reverse();

        String result = robj.ReverseString(str);

        System.out.println("Reverse String is :" + result);

         
    }
}

File: Assignment_35/Program5.c
#include <stdio.h>

void Reverse(char *str)
{
    int i = 0, j = 0;
    char temp;

    for(i = 0; str[i] != '\0'; i++);

    i--;    

    for(j = 0; j < i; j++, i--)
    {
        temp = str[j];
        str[j] = str[i];
        str[i] = temp;
    }
}

int main()
{
    char arr[20];

    printf("Enter string : ");
    scanf("%[^'\n']s", arr);

    Reverse(arr);

    printf("Reversed string is : %s\n", arr);

    return 0;
}